src/app/MessageDef/EventDataIB.cpp

```diff
@@ -242,6 +242,8 @@ CHIP_ERROR EventDataIB::Parser::ProcessEventPath(EventPathIB::Parser & aEventPat
     err = aEventPath.GetEvent(&(aConcreteEventPath.mEventId));
     VerifyOrReturnError(err == CHIP_NO_ERROR, CHIP_ERROR_IM_MALFORMED_EVENT_PATH_IB);
 
+    err = aEventPath.GetIsUrgent(&(aConcreteEventPath.mIsUrgentEvent));
```

Comment:
Why do we want to do this?  The spec is not very clear on this, but IsUrgent is an input, not an output; EventDataIB should never have it present...

Filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/5596 to get this clarified.

Reply:
agree, it should be input